src/data/list/basic.lean

```diff
@@ -798,6 +798,22 @@ begin
   { simp }
 end
 
+lemma mem_of_mem_tail {l : list α} {a : α} (h : a ∈ l.tail) : a ∈ l :=
```

Comment:
Note that we have `tail_subset` later in the same file.

Reply:
Golfed, thanks.